systemwidealerts.md

```diff
@@ -0,0 +1,80 @@
+# System-wide alerts
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint exposes system-wide alerts to be shown as banners in the frontend.
+Only administrators are allowed to create and edit these alerts, while all other users can view them.
+
+This feature will be expanded in the future
+* DSpace currently only supports a single system-wide alert, but the endpoints are designed to accomodate multiple alerts
+* Alerts can specify whether/how user sessions should be restricted while the alert is active, but this is a placeholder as session control has not been implemented yet.
+
+
+## Retrieve all alerts
+**GET /api/system/systemwidealerts**
+This endpoint returns a list of all system-wide alerts.
+
+Status codes:
+* 200 OK - if the operation succeeded
```

Comment:
We should add note that a 403 is returned if a non-admin attempts to access this endpoint

Reply:
401/403 added below

(added `GET /api/system/systemwidealerts/search/active` at the top since it's the most public one)